packages/firestore/src/local/indexeddb_remote_document_cache.ts

```diff
@@ -58,14 +58,23 @@ import { RemoteDocumentChangeBuffer } from './remote_document_change_buffer';
 import { IterateOptions, SimpleDbStore } from './simple_db';
 import { ObjectMap } from '../util/obj_map';
 
-export class IndexedDbRemoteDocumentCache implements RemoteDocumentCache {
+export interface IndexedDbRemoteDocumentCache extends RemoteDocumentCache {
+  // The IndexedDbRemoteDocumentCache doesn't implement any methods on top
+  // of RemoteDocumentCache. This class exists for consistency.
+}
+
+/**
+ * The RemoteDocumentCache for IndexedDb. To construct, invoke
+ * `newIndexedDbRemoteDocumentCache()`.
+ */
+export class IndexedDbRemoteDocumentCacheImpl implements RemoteDocumentCache {
```

Comment:
Should `IndexedDbRemoteDocumentCache` implement `IndexedDbRemoteDocumentCache` instead of `RemoteDocumentCache`?

Reply:
Yes. Thanks for catching.